victoria-metrics/templates/deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}
     {{- if .Values.podAnnotations }}
       annotations:
-    {{ toYaml .Values.podAnnotations | indent 4 }}
+        {{ toYaml .Values.podAnnotations | indent 8 | trim }}
```

Comment:
Thank you for finding this bug, I'd prefer to not use trim. 
let's do it like in all other helm charts in helm/charts, for example
https://github.com/helm/charts/blob/cbd5e811a44c7bac6226b019f1d1810ef5ee45fa/stable/kube-lego/templates/deployment.yaml#L14-L17

basically like
```
diff --git a/victoria-metrics/templates/deployment.yaml b/victoria-metrics/templates/deployment.yaml
index f08192c..c204afe 100644
--- a/victoria-metrics/templates/deployment.yaml
+++ b/victoria-metrics/templates/deployment.yaml
@@ -21,10 +21,10 @@ spec:
       labels:
         app.kubernetes.io/name: {{ include "victoria-metrics.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
-    {{- if .Values.podAnnotations }}
+{{- if .Values.podAnnotations }}
       annotations:
-    {{ toYaml .Values.podAnnotations | indent 4 }}
-    {{- end }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
     spec:
       containers:
         - name: {{ .Chart.Name }}
```

Reply:
ok. I'm apply this patch in pull request.
Sorry for waiting.